### ISSUE TITLE
ECS: Failed to extract AWS Session Manager Plugin

### ISSUE THREAD

**Issue by soleson** [aws/aws-toolkit-jetbrains#3130]
**Describe the bug**
I am unable to connect to a running AWS Fargate Task Container as the Session Manager Plugin fails to install. I'm assuming that the issue might be related to the executing user being unprivileged, although the User does have permission to `sudo`. Phpstorm is installed inside a WSL2 implementation of Ubuntu.

**To reproduce**
1. Install AWS Toolkit.
2. Verify AWS resources are accessible via the IDE.
3. Choose an ECS Service Task that has the Execute Command enabled already.
4. Attempt to "Open Interactive Shell..." to one of the Containers within that Task.
5. Leave the default values unchanged and click Execute.
6. The IDE returns an error: "ECS Execute Command Failed", "Failed to install AWS Session Manager Plugin"
7. The IDE log shows the following errors:
    ```
     2022-04-15 10:20:26,163 [1510786]   WARN - s.a.t.j.s.e.e.OpenShellInContainerDialog - Failed to start interactive shell
    java.lang.IllegalStateException: Failed to install AWS Session Manager Plugin
    	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager.installTool(DefaultToolManager.kt:213)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager.access$installTool(DefaultToolManager.kt:44)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager$getOrInstallTool$1.invoke(DefaultToolManager.kt:75)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager$getOrInstallTool$1.invoke(DefaultToolManager.kt:70)
	at software.aws.toolkits.jetbrains.utils.ThreadingUtilsKt.runUnderProgressIfNeeded(ThreadingUtils.kt:30)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager.getOrInstallTool(DefaultToolManager.kt:70)
	at software.aws.toolkits.jetbrains.core.tools.ToolManagerKt.getOrInstallTool(ToolManager.kt:77)
	at software.aws.toolkits.jetbrains.services.ecs.exec.EcsExecUtils.createCommand(EcsExecUtils.kt:233)
	at software.aws.toolkits.jetbrains.services.ecs.exec.OpenShellInContainerDialog.runExecCommand(OpenShellInContainerDialog.kt:92)
	at software.aws.toolkits.jetbrains.services.ecs.exec.OpenShellInContainerDialog.access$runExecCommand(OpenShellInContainerDialog.kt:36)
	at software.aws.toolkits.jetbrains.services.ecs.exec.OpenShellInContainerDialog$doOKAction$1.invokeSuspend(OpenShellInContainerDialog.kt:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
    Caused by: java.lang.IllegalStateException: Failed to extract AWS Session Manager Plugin
    STDOUT:
    STDERR:dpkg-deb (subprocess): failed to create directory: No such file or directory
    dpkg-deb: error: tar subprocess returned error exit status 2
    dpkg-deb (subprocess): decompressing archive member: internal gzip write error: Broken pipe
    dpkg-deb (subprocess): cannot copy archive member from '/tmp/session-manager-plugin5280829067625868679/session- manager-plugin.deb' to decompressor pipe: failed to write (Broken pipe)

	at software.aws.toolkits.jetbrains.services.ssm.SsmPlugin.runInstall(SsmPlugin.kt:113)
	at software.aws.toolkits.jetbrains.services.ssm.SsmPlugin.installVersion(SsmPlugin.kt:65)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager.performInstall$lambda-9(DefaultToolManager.kt:243)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.computeWithLockAndCheckingCanceled(ProgressIndicatorUtils.java:337)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager.performInstall(DefaultToolManager.kt:232)
	at software.aws.toolkits.jetbrains.core.tools.DefaultToolManager.installTool(DefaultToolManager.kt:208)
	... 19 more
    ```
**Expected behavior**
The AWS Session Manager Plugin should complete installation.

**Your Environment**
- OS: WSL2 Ubuntu (20.04.4 LTS)
- JetBrains product: Phpstorm
- JetBrains product version: 2022.1
- AWS Toolkit version: 1.42-221
- SAM CLI version: 1.46.0
- JVM/Python version: 3.8.10

**Comment by soleson** [comment]
Although, the root issue is not resolved, I was able to work around it by installing AWS Session Manager independently and specifying the path to the AWS Session Manager within the project settings:

Settings > Tools > AWS > External Tools > AWS Session Manager Plugin

To determine the path of the AWS Session Manager binary (after successful installation), you should be able to execute the following in the terminal (Assuming the OS is linux):
```bash
which session-manager-plugin
```

**Comment by rli** [comment]
Released with `1.47`
